client/css/style.css

```diff
@@ -268,6 +268,16 @@ kbd {
 #sidebar .chan.lobby::before,
 #chat .lobby .title::before { content: "\f0a0"; /* http://fontawesome.io/icon/hdd-o/ */ }
 
+#sidebar .not-secure .chan.lobby::before {
+	content: "\f09c"; /* http://fontawesome.io/icon/unlock/ */
+	color: #ff1744;
+}
+
+#sidebar .not-connected .chan.lobby::before {
+	content: "\f1e6"; /* http://fontawesome.io/icon/plug/ */
```

Comment:
Not a huge fan of this icon. I seems pretty common to see [`chain-broken`](https://fontawesome.com/v4.7.0/icon/chain-broken/) for broken connections/links (even nicer with [its v5 equivalent](https://fontawesome.com/icons/unlink?style=solid)), or we could go with something more generic like [`ban`](https://fontawesome.com/v4.7.0/icon/ban/) or [`exclamation`](https://fontawesome.com/v4.7.0/icon/exclamation/).

Reply:
Yeah the icons are crap because I couldn't find anything good in FA.

For example, for insecure I'd really like to have a lock with strike through line, not whatever it is now.

`unlink` is probably the best we have for disconnected.